### PR TITLE
Avoid the deletion of still used address in order history

### PR DESCRIPTION
**Pull request by julienbourdeau**
Until this PR is merged, if a customer edit an address, all order with this address will be affected. (bug introduce between 1.6 and 1.7)

This PR fixes this.

### CODE REVIEW

File: tests/selenium/specs/customer/addresses.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 var fixtures = require('../../fixtures');
 var _        = require('underscore');
 
-describe('Customer account: Addresses', function () {
+describe.only('Customer account: Addresses', function () {
```
Comment (maximebiloe) [inline tests/selenium/specs/customer/addresses.js]: Can you remove the `only`, please ?

Comment (julienbourdeau) [inline tests/selenium/specs/customer/addresses.js]: :cry:

Comment (nihco2) [inline tests/selenium/specs/customer/addresses.js]: :scream:

**Comment by maximebiloe** [comment]
Could you make sure everything is fine? Tests seem to be broken :confused:

**Comment by mickaelandrieu** [comment]
Good job @julienbourdeau, thanks for the tests improvements ;)

**Pull request merged**